doc/tutorials.rst

```diff
@@ -0,0 +1,89 @@
+ .. role:: html(raw)
+   :format: html
+
+.. _New_Users:
+
+Tutorials
+=========
+
+The following tutorials introduce the core PennyLane concepts, including QNodes,
+plugins, and devices, via simple and easy to follow examples.
+
+
+:html:`<h3>Learn the PennyLane basics</h3>`
+
+The following tutorials introduce the core PennyLane concepts, including QNodes,
+plugins, and devices, via simple and easy-to-follow examples.
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to rotate a qubit.
+    :figure: ../examples/figures/bloch.png
+    :description: :ref:`qubit_rotation`
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning to tune a beamsplitter.
+    :figure: ../examples/figures/gaussian_transformation.png
+    :description: :ref:`gaussian_transformation`
+
+.. customgalleryitem::
+    :tooltip: Use quantum machine learning in a multi-device quantum algorithm.
+    :figure: ../examples/figures/photon_redirection.png
+    :description: :ref:`plugins_hybrid`
+
+.. customgalleryitem::
+    :tooltip: Multiple expectation values, Jacobians, and keyword arguments.
+    :description: :ref:`advanced_features`
```

Comment:
I was confused - this is the "basics" section, but there is advanced usage? (Or was this discussed and resolved somewhere?)

Reply:
I think basics is the wrong word - the first group of tutorials are all focused on teaching PennyLane, with the QML aspect taking a backseat (i.e., the simple qubit rotation as a backdrop for introducing PL formalism).

The second group of tutorials instead assume knowledge of PennyLane, and instead delve deeper into QML concepts and algorithms.

Any suggestions for alternative tutorial section names?

Reply:
How about just "Learn PennyLane" and "Quantum machine learning algorithms"

Reply:
> How about just "Learn PennyLane" and "Quantum machine learning"

sounds great. How about "QML with PennyLane" for the second one? QML in itself sounds very general.

Reply:
done!